client/src/App.tsx

```diff
@@ -1520,7 +1519,8 @@ function App() {
 			} else {
 				setIsInvalidRoom(true);
 				if (result.message) {
-					setModalErrorMessage(result.message);
+					//setModalErrorMessage(result.message);
+					setInvalidRoomMessage(result.message);
```

Comment:
Because of const address = req.user ? req.user.payload.publicAddress.toLowerCase() : ""; we get addresss slow and first couple of request returns gives empty contract addresses. Thus even authorized gets error message at first. Also even we sent modalErrorMessage unauthorized users won't see it because of line 2057: if (isInvalidRoom) {
  return <div>Invalid room {roomId} : {invalidRoomMessage} <MetamaskSection /> </div>;

}